<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>479</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>479</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Profile Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>811</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutMain">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="1" rowspan="5">
      <widget class="QGroupBox" name="groupBoxP">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Personal info</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>421</width>
          <height>321</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayoutP">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="2" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout3n">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Patronymic</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit3n"/>
           </item>
          </layout>
         </item>
         <item row="3" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayoutBD">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Birth Date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxD">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>31</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxM"/>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxYr">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>1875</number>
             </property>
             <property name="maximum">
              <number>2025</number>
             </property>
             <property name="value">
              <number>1950</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="2">
          <layout class="QHBoxLayout" name="horizontalLayoutS">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Sex</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxS">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayoutEm">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="minimumSize">
              <size>
               <width>113</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Emaill</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditEm">
             <property name="maximumSize">
              <size>
               <width>220</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout1n">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>1st Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit1n"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayout2n">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>2nd Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit2n"/>
           </item>
          </layout>
         </item>
         <item row="5" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayoutPh">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="minimumSize">
              <size>
               <width>113</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Phone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditPh">
             <property name="maximumSize">
              <size>
               <width>220</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0" colspan="3">
          <layout class="QHBoxLayout" name="horizontalLayoutMIId">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Medical Insurance</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMI">
             <property name="maximumSize">
              <size>
               <width>220</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="QGroupBox" name="groupBoxPh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>167772</height>
        </size>
       </property>
       <property name="title">
        <string>Photo</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>221</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutP">
         <property name="leftMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QGraphicsView" name="graphicsView">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>200</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadButton">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="3" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBoxDi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>300</height>
        </size>
       </property>
       <property name="title">
        <string>Diagnosis info</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>212</width>
          <height>156</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayoutDi">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>-1</number>
         </property>
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEditA">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>135</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Age at Diagnosis</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Death Rate </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="lineEditDR">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QFrame" name="frame_2">
           <property name="minimumSize">
            <size>
             <width>45</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="4">
          <widget class="QTextEdit" name="textEdit">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>135</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Diagnosis:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <zorder>label_9</zorder>
       <zorder>lineEditDR</zorder>
       <zorder>label_8</zorder>
       <zorder>lineEditDi</zorder>
       <zorder>label_5</zorder>
       <zorder>lineEditA</zorder>
       <zorder>label_5</zorder>
       <zorder>lineEditA</zorder>
      </widget>
     </item>
     <item row="0" column="2" rowspan="4">
      <widget class="QGroupBox" name="groupBoxPwE">
       <property name="maximumSize">
        <size>
         <width>212</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>List of Pictures with Evedences</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>201</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutPwE">
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAEP">
           <property name="text">
            <string>Add/Edit</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QGroupBox" name="groupBoxSU">
       <property name="title">
        <string>Save/Update</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>191</width>
          <height>66</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutSU">
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="maximumSize">
            <size>
             <width>215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="maximumSize">
            <size>
             <width>215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEdit"/>
   <addaction name="actionAdd"/>
   <addaction name="actionDel"/>
   <addaction name="actionPhoto"/>
   <addaction name="actionPersonal"/>
   <addaction name="actionDiagnosis"/>
   <addaction name="actionAddPicture"/>
   <addaction name="actionSave"/>
   <addaction name="actionUpdate"/>
  </widget>
  <action name="actionEdit">
   <property name="icon">
    <iconset>
     <normalon>:/MyFiles/icons/edit_ico.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit Find Profile</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset>
     <normalon>:/MyFiles/icons/add_info_ico.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add Profile</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset>
     <normalon>:/MyFiles/icons/del_info_ico.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
   <property name="toolTip">
    <string>Delete Current</string>
   </property>
  </action>
  <action name="actionPhoto">
   <property name="icon">
    <iconset>
     <normalon>:/MyFiles/icons/photo_ico.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Photo</string>
   </property>
   <property name="toolTip">
    <string>Profile Photo</string>
   </property>
  </action>
  <action name="actionDiagnosis">
   <property name="icon">
    <iconset>
     <normalon>:/MyFiles/icons/diagnosis_ico.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Diagnosis</string>
   </property>
   <property name="toolTip">
    <string>Input diagnosis</string>
   </property>
  </action>
  <action name="actionPersonal">
   <property name="icon">
    <iconset>
     <normalon>:/MyFiles/icons/personal_ico.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Personal</string>
   </property>
   <property name="toolTip">
    <string>Profile personals </string>
   </property>
  </action>
  <action name="actionAddPicture">
   <property name="icon">
    <iconset>
     <normalon>:/MyFiles/icons/add_image_ico.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>AddPicture</string>
   </property>
   <property name="toolTip">
    <string>Add Picture</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/MyFiles/icons/save_ico.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Current Changes</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset>
     <normalon>:/MyFiles/icons/update_ico.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Load Changes From DB</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
